kornia/augmentation/augmentation.py

```diff
@@ -68,10 +68,8 @@ class RandomHorizontalFlip(GeometricAugmentationBase2D):
         - Input: :math:`(C, H, W)` or :math:`(B, C, H, W)`, Optional: :math:`(B, 3, 3)`
         - Output: :math:`(B, C, H, W)`
 
-    Note:
-        Input tensor must be float and normalized into [0, 1] for the best differentiability support.
-        Additionally, this function accepts another transformation tensor (:math:`(B, 3, 3)`), then the
-        applied transformation will be merged int to the input transformation tensor and returned.
```

Comment:
@shijianjian I removed this once from everywhere since it's a repeating pattern across the augmentations and should be placed somewhere in general documentation.